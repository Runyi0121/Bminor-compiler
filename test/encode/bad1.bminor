"h,a,p,p,y\0x00c,o,d,i,n,g"