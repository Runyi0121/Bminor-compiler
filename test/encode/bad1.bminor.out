--encode ./test/encode/bad1.bminor
There is an error in the original string.
